examples/features/annotated-namespace/README.md

```diff
@@ -0,0 +1,63 @@
+# Test kernel to kernel connection with annotated namespace
+
+This example shows that NSM annotations applied to namespace will be applied to the pods within this namespace.  
+
+NSC and NSE are using the `kernel` mechanism to connect to its local forwarder.
+
+
+## Requires
+
+Make sure that you have completed steps from [basic](../../basic) or [memory](../../memory) setup.
+
+## Run
+
+Create test namespace and deploy NSE:
+```bash
+kubectl apply -k https://github.com/networkservicemesh/deployments-k8s/examples/features/annotated-namespace?ref=76c014f23bf7027dbd520ee957aa6c0135184dc7
+```
+
+Wait for NSE to be ready:
+```bash
+kubectl wait --for=condition=ready --timeout=1m pod -l app=nse-kernel -n ns-annotated-namespace
+```
+
+Annotate namespace with NSM annotation:
+```bash
+kubectl annotate --overwrite ns ns-annotated-namespace networkservicemesh.io=kernel://annotated-namespace/nsm-1
```

Comment:
Why do we need an option `--overwrite ns` ?

Reply:
we don't, option is removed.